src/AWS.Deploy.Common/Recipes/Validation/RecipeValidators/BeanstalkManagedUpdatesHealthReportingValidator.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AWS.Deploy.Common.Recipes.Validation
+{
+    public class BeanstalkManagedUpdatesHealthReportingValidator : IRecipeValidator
```

Comment:
Instead of having the validator should we design the recipe to not even allow an invalid combination. Make `EnhancedHealthReportingEnabled` depend on `ManagedActionsEnabled` being false. Then in the CDK code enable enhanced health if either `ManagedActionsEnabled` is enabled or `EnhancedHealthReportingEnabled` is enabled.

Reply:
Agreed

Reply:
Done